development.ini.sample

```diff
@@ -60,6 +60,11 @@ mailman.url = https://mailman.mines.edu/mailman/admin/acmx
 mailman.pipermail.url = https://mailman.mines.edu/pipermail/acmx
 mailman.secret = getthisfromjack
 
+# Social Links
+social_links = github,facebook
```

Comment:
Why not just see which keys are defined? No need for this config option...

Reply:
I asked for feedback on this ages ago: https://github.com/ColoradoSchoolOfMines/mozzarella/pull/37#issue-159731128.

I couldn't find any way to check which keys are defined. Any suggestions would be appreciated.

Reply:
> I couldn't find any way to check which keys are defined. Any suggestions would be appreciated.

``tg.config`` is like a dictionary (in fact, it inherits from ``collections.abc.MutableMapping``), so you can just call ``.keys()`` on it.

Reply:
Want me to fix this or are you working on it?

Reply:
I'm on it. Commit incomming...